evidence/ermalkaleci/2024-06-21.md

```diff
@@ -0,0 +1,32 @@
+# Retention at Rank 1
+
+|                  |                                 |
+| ---------------- | ------------------------------- |
+| **Report Date**  | Date of submission (2024/06/21) |
+| **Submitted by** | Ermal Kaleci                    |
+
+
+## Member details
+
+- Matrix username: @ermalkaleci:matrix.org
+- Polkadot address: 14iKbZws1fjJ6TH27yoRq6KeeVNof83VmxUBN2W2udQVBe5o
+- Current rank: 1
+- Date of initial induction: 2024-04-02
```

Comment:
Ermal was [inducted](ipfs://bafybeibifzurkx5a676ja4fgvyg2t6y2muqxch4aj5vbit65yzi35d6wim/#/extrinsics/3206006-2) on 17th February 2024. 
Please update the form to reflect this date.

Reply:
@anaelleltd I became rank 1 on 2024-04-02
https://collectives.subsquare.io/fellowship/referenda/82?tab=timeline

Reply:
I guess you're referring to [this](https://collectives.statescan.io/#/extrinsics/3206006-2)

Reply:
@anaelleltd do I need to resubmit evidences?

Reply:
> @anaelleltd do I need to resubmit evidences?
We only need the induction date on the form, not the date of the last promotion.
Yes, please resubmit on Subsquare when the PR is updated.

Reply:
Done.